src/PowerShellEditorServices.Host/CodeLens/PesterCodeLensProvider.cs

```diff
@@ -33,6 +40,38 @@ public PesterCodeLensProvider(EditorSession editorSession)
         {
             _editorSession = editorSession;
             _symbolProvider = new PesterDocumentSymbolProvider();
+
+            DeterminePesterVersion();
+        }
+
+        /// <summary>
+        /// Used to determine the value of <see cref="_pesterV4_6_0_OrHigherAvailable"/> as a background task.
+        /// </summary>
+        private void DeterminePesterVersion()
+        {
+            Task.Run(() =>
+            {
+                using (var powerShell = System.Management.Automation.PowerShell.Create())
+                {
+                    powerShell.AddCommand("Get-Module")
+                              .AddParameter("ListAvailable")
+                              .AddParameter("Name", "Pester");
+                    var result = powerShell.Invoke();
```

Comment:
```suggestion
                    IEnumerable<PSObject> result = powerShell.Invoke();
```

Reply:
It needs to be an `ICollection` at least because we call `.Count` on it, which is better (faster) than calling `.Any()` on it.